src/include/search.h

```
@@ -40,6 +42,10 @@ extern "C" {
 extern struct sratom_str *sr_atoms;
 extern struct search_str *sr_search;
 extern char *sr_project;
+extern int sr_amoeba_eval_limit;
```

Clang (14.0.6)

[warning] variable 'sr_amoeba_eval_limit' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)